src/main/g8/pom.xml

```diff
@@ -41,8 +49,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.5.1</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
not sure, but I think those should be replaced by
```
<release>11</release>
```

and in samples like this we can probably promote 17 more? (that might need change in CI also)

Reply:
Ok, let's go with 17 across the quickstarts then

Reply:
Had to bump gradle from ancient version to 7.6 for JDK 17 support